vllm/engine/arg_utils.py

```diff
@@ -201,10 +201,12 @@ def add_cli_args(
                             help='model context length. If unspecified, '
                             'will be automatically derived from the model.')
         # Parallel arguments
-        parser.add_argument('--worker-use-ray',
-                            action='store_true',
-                            help='use Ray for distributed serving, will be '
-                            'automatically set when using more than 1 GPU')
+        parser.add_argument(
+            '--worker-use-ray',
+            action=argparse.BooleanOptionalAction,
```

Comment:
@zhuohan123 it allows having a default to a boolean arg of `None` so that we can differentiate between not set and explicitly set to false.

It looks like this may have been introduced in python 3.9 though, so I guess may need to be changed anyhow.